src/applications/disability-benefits/all-claims/pages/contactInformation.js

```diff
@@ -9,18 +9,19 @@ import PhoneNumberWidget from 'us-forms-system/lib/js/widgets/PhoneNumberWidget'
 import ReviewCardField from '../components/ReviewCardField';
 
 import {
-  PrimaryAddressViewField,
-  ForwardingAddressViewField,
   contactInfoDescription,
   contactInfoUpdateHelp,
+  forwardingAddressDescription,
+  ForwardingAddressViewField,
   phoneEmailViewField,
+  PrimaryAddressViewField,
 } from '../content/contactInformation';
 
 import {
-  validateZIP,
+  isInFuture,
   validateMilitaryCity,
   validateMilitaryState,
-  isInFuture,
+  validateZIP,
```

Comment:
Heh, out of curiosity, do you have an import alphabetizer plugin for your editor or something? :smile:

Reply:
I have a plugin but it's manually activated—I sorted these to make it easier to scan 🔡